ClientApp/src/Pages/DonatiePage/DonatiePagina.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+
+
+export class DonatiePagina extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            donatieBedrag: ""
+        };
+
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(e) {
+        this.setState({ [e.target.name]: e.target.value });
+    }
+
+    handleSubmit(e) {
+        e.preventDefault();
+
+        const gebruiker = JSON.parse(sessionStorage.getItem("gebruiker"));
```

Comment:
we moeten alleen wat anders voor dei sessionStorege 'gebruiker'. Dat moet wsl verandert worden in een fetch naar de api om de bezoeker info op te halen.